storybook/stories/API/props/ChartProps.ts

```diff
@@ -181,6 +181,16 @@ toggling between multiple dataKey.`,
       category: 'General',
     },
   },
+  stepRatioControl: {
+    description: 'The value to control the range of y axis domain',
```

Comment:
I'm not sure if this is enough description for people to understand what this is and what it accomplishes